package.json

```diff
@@ -13,6 +13,9 @@
   "engines": {
     "node": ">=8"
   },
+  "resolutions": {
+    "simple-html-tokenizer": "0.5.7"
+  },
```

Comment:
It is bad idea

Reply:
wait, explaining

Reply:
@fisker we need find a problem and report about this and wait a fix, no need change resolutions

Reply:
see comment bellow